conf/spark-env.sh

```diff
@@ -12,7 +12,7 @@ mkdir -p "${HADOOP_CONF_DIR}"
 [ -f "${MESOS_SANDBOX}/hdfs-site.xml" ] && cp "${MESOS_SANDBOX}/hdfs-site.xml" "${HADOOP_CONF_DIR}"
 [ -f "${MESOS_SANDBOX}/core-site.xml" ] && cp "${MESOS_SANDBOX}/core-site.xml" "${HADOOP_CONF_DIR}"
 
-
+SPARK_WORKING_DIR=${PWD}
```

Comment:
Is the CWD explicitly set? I know that in TensorFlow I saw some undefined behaviour if the `WORKDIR` was not set in a docker image.

Reply:
I don't think its explicitly set but it looks like [this spark script](https://github.com/mesosphere/spark/blob/custom-branch-2.2.1-X/bin/load-spark-env.sh#L20) finds this file and loads all its envvars for every spark job so the PWD would at least point inside Spark?

Reply:
I don't see how that script changes the current working directory though? I was mainly asking because this seems to introduce an envvar that is not related to the metrics and is not used elsewhere.

Dropping this Envvar and using relative paths in the templating below would have the same effect.